Sources/Tracking/CustomerIO.swift

```diff
@@ -64,6 +75,14 @@ public extension CustomerIOInstance {
     ) {
         identify(identifier: identifier, body: body, onComplete: onComplete, jsonEncoder: jsonEncoder)
     }
+
+    func track(
+        name: String,
+        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void,
+        jsonEncoder: JSONEncoder? = nil
+    ) {
```

Comment:
When using the SDK in Remote Habits, I noticed all 4 of the options that we give customers for using this function:

![2021-09-16 at 9 52 AM](https://user-images.githubusercontent.com/2041082/133634970-ba0ef313-abb5-43eb-9c9c-9753f0eaa892.jpeg)

We want to try and keep `onComplete` always as the last parameter because it's a lambda and that's common practice to have lambda be last. 

Notice in the screenshot 1 of the 4 options doesn't have `onComplete` as the last option. I believe this is the function that makes that so.

Reply:
should fix that for `identify` then as well I think it doesn't fit that pattern
<img width="364" alt="Screen Shot 2021-09-16 at 4 10 02 PM" src="https://user-images.githubusercontent.com/661424/133696918-4d6e3d5a-f321-4633-ad24-ced9962862a0.png">